base_latt: 52.220395
base_long: 21.010866